### PR TITLE
Cherry-pick #19568 to 7.x: [Filebeat] Add default_fields: false to fields.yml in aws module

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
Cherry-pick of PR #19568 to 7.x branch. Original message: 

I saw error about too many fields in Filebeat when running `mage build unitTest`:
```
Package: github.com/elastic/beats/v7/x-pack/filebeat
Test:    TestTemplate/default_field_length
template.go:75: Too many fields (1002>1000) in Filebeat index templatesettings.index.query.default_field for comfort. By default Elasticsearch has a limit of 1024 fields in a query so we need to keep the number of fields below 1024. Adding 'default_field: false' to fields or groups in a fields.yml can be used to reduce the number of text/keyword fields that end up in default_field.
```
So I went to check if I have `default_field: false` added for filesets under `aws`. This PR is to add the missing `default_field: false` for `aws` module.

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -227,6 +227,8 @@ field. You can revert this change by configuring tags for the module and omittin
 - Fix `netflow` module to support 7 bytepad for IPFIX template. {issue}18098[18098]
 - Fix improper nesting of session_issuer object in aws cloudtrail fileset. {issue}18894[18894] {pull}18915[18915]
 - Fix date and timestamp formats for fortigate module {pull}19316[19316]
+- Fix memory leak in tcp and unix input sources. {pull}19459[19459]
```
Comment (jsoriano) [inline CHANGELOG.next.asciidoc:230]: Not related.

Comment (kaiyan-sheng) [reply CHANGELOG.next.asciidoc]: Yes thanks!

**Comment by ycombinator** [comment]
CHANGELOG file has conflicts :(

**Comment by kaiyan-sheng** [comment]
@ycombinator I guess we have a lot of cherrypicks going on right now 😄 Just resolved the conflict! Thanks!

**Pull request merged**